boards/phytec/phyboard_lyra/phyboard_lyra_am6234_m4_defconfig

```diff
@@ -5,6 +5,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# Don't fill gaps to increase linking time
```

Comment:
Consider rewording this, it decreases the post build hex file generation time.
But maybe something about non-contiguous sections producing unwieldy binary and gap filled hex files would be clearer

Reply:
Thanks! Is the new wording better?

Reply:
Not really, I wasn't very clear...

The increased build time is due to a post build step generating a 7GB hex file.

This is due to the output image having non-contiguous sections (~2.5GB apart):
```
  [ 1] rom_start         PROGBITS        00000000 0000d8 000140 00 WAX  0   0  4
  ...
  [36] .resource_table   PROGBITS        9cc00000 00decc 000010 00  WA  0   0  4
```
Text/rodata/data/bss in sram0 (0x0).
openamp resource table in ddr0 (0x9cc00000).

Maybe something like this?

> The output image has non-contiguous sections, disable
gap-fill to avoid very large hex files and long build times

The output binary is already disabled for this same reason too:
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/ti/k3/am6x/Kconfig.defconfig#L42

Reply:
this explanation was very helpful, thanks. I just enabled the CONFIG for all AM6 M4s since we cannot flash a hex anyways (at least right now).